e2e/pull/pull.go

```diff
@@ -268,26 +263,69 @@ func getImageNameFromURI(imgURI string) string {
 	return uri.GetName(imgURI)
 }
 
-func testPullCmd(t *testing.T) {
+func (c *ctx) setup(t *testing.T) {
```

Comment:
this one is the only change that is bigger than it needs to be. There was a bunch of code in the set up portion of the test, so I gave it a name. The code is unchanged.